### PR TITLE
CSE-277 and copyright

### PR DESCRIPTION
**Pull request by dbukhmastov**
- code correction accordind to CSE-277 comment
- adding the copyrights

### CODE REVIEW

File: HerokuApp/index.js [hunk 1]
```diff
@@ -112,9 +115,13 @@ function sUpdate(query, received){
                     }
                     let sql;
                     if (result.rows.length > 0) {
-                        sql = "UPDATE " + TABLE_NAMES['fridge'] + " SET " + changes.join(',') + " WHERE deviceId__c='" + query['deviceId__c'] + "'";
+                        sql = `UPDATE ${TABLE_NAMES['fridge']} SET ${changes.join(',')} WHERE deviceId__c='${query['deviceId__c']}'`;
                     } else {
-                        sql = "INSERT INTO " + TABLE_NAMES['fridge'] + " (deviceId__c, door__c, temperature__c, humidity__c, ts__c) VALUES ('" + query['deviceId__c'] + "', '" + (query['door__c'] === undefined ? 'closed' : query['door__c']) + "', " + (query['temperature__c'] === undefined ? 0 : query['temperature__c']) + ", " + (query['humidity__c'] === undefined ? 0 : query['humidity__c']) + ", '" + (query['ts__c'] === undefined ? '2000-01-01T00:00:00Z' : query['ts__c']) + "')";
+						let door__c = query['door__c'] === undefined ? 'closed' : query['door__c'];
```
Comment (ppetrosh) [inline HerokuApp/index.js]: tabs? please use spaces instead

File: HerokuApp/index.js [hunk 2]
```diff
@@ -235,8 +242,8 @@ function sCase(query, received){
  *     client : object - postgres client
  *     done : function - postgres on done fuction
  */
-function _sCase(query, resolve, reject, client, done){
-    let sql = "INSERT INTO " + TABLE_NAMES['case'] + " (Subject, Description, Related_Fridge__r__deviceId__c) VALUES ('" + query['Subject'] + "', '" + query['Description'] + "', '" + query['Related_Fridge__r']['DeviceId__c'] + "') RETURNING Id";
+function sCaseExecute(query, resolve, reject, client, done){
```
Comment (ppetrosh) [inline HerokuApp/index.js]: insertCase?

**Comment by ppetrosh** [comment]
@dbukhmastov did you verify these changes on heroku and the device sides? I guess I will need to do so too...

**Comment by dbukhmastov** [comment]
Yes. Device, agent and JS codes have been tested within 1 hour. So I think it is'n needed to do this again.

**Review by ppetrosh — APPROVED** [review]
Thank you, Dmitry!

**Pull request merged**